src/plans/D3MPlanBase.t.sol

```diff
@@ -88,7 +88,7 @@ contract D3MPlanBaseTest is DSTest {
             address(bytes20(uint160(uint256(keccak256("hevm cheat code")))))
         );
 
-        dai = DaiLike(address(123));
+        dai = DaiLike(address(0x123));
```

Comment:
It's pretty common we do `address(123)` in general, it just means something random.
I mean not against to the change, but don't think it is a fix at all.

Reply:
yeah, same as above, was meant to be a nit

Reply:
i'd revert then for consistency with our other dummy addresses

Reply:
e76057a